tests/test_rpc.py

```
@@ -0,0 +1,62 @@
+    
+    async with aiohttp.ClientSession() as session:
+        payload = {"jsonrpc": "2.0", "method": "mc_blockNumber", "id": 1}
+        async with session.post(f"http://127.0.0.1:{port}/", json=payload) as resp:
+            assert resp.status == 200
+            data = await resp.json()
+            assert data["result"] == 0
+            assert data["id"] == 1
+
+@pytest.mark.anyio
+async def test_rpc_getBlockByNumber(rpc_server):
+    server, port, chain, mempool = rpc_server
```

Ruff (0.15.15)

[warning] Unpacked variable `server` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)